### PR TITLE
Fix Majority of the Typos in `transformers[en]` Documentation

### LINKED ISSUE #33349: List of Typos in the Transformers[EN] Docs
Hey There, I went through the entire `transformers[en]` documentation, and have found a list of potential typos.

I have categorized these typos as follows:

**_(1)General Typos_** - Spelling errors throughout the text.
**_(2)Code Snippet Errors_** - Missing or incorrectly defined code snippets.
**_(3)Naming Inconsistencies_** - Inconsistent use of class/library names.
**_(4)Broken Links_** - Hyperlinks that do not lead to the expected pages.
**_(5)Omissions in Text_** - Sentences that are missing words, affecting clarity.
**_(6)Punctuation Errors_** - Incorrect or missing punctuation marks.
**_(7)Minor Text Issues_** - Slight wording errors.
**_(8)Formatting Issues_** - Text that should be in code blocks but isn't.

I have listed the Tables, for the same as follows:
**_(1)General Typos:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 1      | https://huggingface.co/docs/transformers/main/en/pr_checks            | insted of                                                                            | instead of                                                                                                  |
| 2      | https://huggingface.co/docs/transformers/main/en/quantization/eetq    | relase                                                                               | release                                                                                                     |
| 3      | https://huggingface.co/docs/transformers/main/en/quantization/fbgemm_fp8 | nighlty                                                                             | nightly                                                                                                     |
| 4      | https://huggingface.co/docs/transformers/main/en/quantization/quanto  | versatible, becnhmark, benchamarks                                                   | versatile, benchmark, benchmarks                                                                            |
| 5      | https://huggingface.co/docs/transformers/main/en/quantization/torchao | quantizd                                                                             | quantized                                                                                                   |
| 6      | https://huggingface.co/docs/transformers/main/en/model_doc/falcon_mamba | architecutre                                                                        | architecture                                                                                                |
| 7      | https://huggingface.co/docs/transformers/main/en/model_doc/hiera      | contibution                                                                          | contribution                                                                                                |
| 8      | https://huggingface.co/docs/transformers/main/en/model_doc/jamba      | Presequities                                                                         | Prerequisites                                                                                               |
| 9      | https://huggingface.co/docs/transformers/main/en/model_doc/matcha     | faste convergence                                                                    | faster convergence                                                                                          |
| 10     | https://huggingface.co/docs/transformers/main/en/model_doc/mbart      | extendeding                                                                          | extending                                                                                                   |
| 11     | https://huggingface.co/docs/transformers/main/en/model_doc/seggpt     | segmetantion_maps                                                                    | segmentation_maps                                                                                           |
| 12     | https://huggingface.co/docs/transformers/main/en/generation_strategies | alternativelly                                                                      | alternatively                                                                                               |

**_(2)Code Snippet Errors:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 13     | https://huggingface.co/docs/transformers/main/en/peft                 | output                                                                               | output_disabled                                                                                             |
| 14     | https://huggingface.co/docs/transformers/main/en/serialization        | tranformers.onnx                                                                     | transformers.onnx                                                                                           |
| 15     | https://huggingface.co/docs/transformers/main/en/testing              | tf.random.set_seed(seed)                                                             | import tensorflow as tf tf.random.set_seed(seed)                                                            |
| 16     | https://huggingface.co/docs/transformers/main/en/torchscript          | torch.neuron.trace(model, [token_tensor, segments_tensors])                           | torch.neuron.trace(model, [tokens_tensor, segments_tensors])                                                |
| 17     | https://huggingface.co/docs/transformers/main/en/tasks/image_to_image | from transformers import pipeline                                                    | from transformers import pipeline import torch                                                              |
| 18     | https://huggingface.co/docs/transformers/main/en/model_doc/mms        | uromaize                                                                             | uromanize                                                                                                   |
| 19     | https://huggingface.co/docs/transformers/main/en/model_doc/vits       | uromaize                                                                             | uromanize                                                                                                   |
| 20     | https://huggingface.co/docs/transformers/main/en/tasks/monocular_depth_estimation | numpy.ndarry                                                                       | numpy.ndarray                                                                                               |
| 21     | https://huggingface.co/docs/transformers/main/en/main_classes/logging | one of the following: debug, info, warning, error, critical.                         | one of the following: debug, info, warning, error, critical, fatal.                                          |
| 22     | https://huggingface.co/docs/transformers/main/en/main_classes/output  | last_hidden_states                                                                   | last_hidden_state                                                                                           |

**_(3)Naming Inconsistencies:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 23     | https://huggingface.co/docs/transformers/main/en/perf_train_gpu_one   | Torchscript                                                                           | TorchScript                                                                                                 |
| 24     | https://huggingface.co/docs/transformers/main/en/model_doc/oneformer  | Oneformer                                                                             | OneFormer                                                                                                   |
| 25     | https://huggingface.co/docs/transformers/main/en/model_doc/xlnet      | TFLNetForMultipleChoice                                                               | TFXLNetForMultipleChoice                                                                                    |
| 26     | https://huggingface.co/docs/transformers/main/en/model_doc/camembert  | TFCamembertForCasualLM                                                                | TFCamembertForCausalLM                                                                                      |
| 27     | https://huggingface.co/docs/transformers/main/en/quantization/hqq     | Pytorch                                                                               | PyTorch                                                                                                     |
| 28     | https://huggingface.co/docs/transformers/main/en/main_classes/logging | logging's captureWarning method                                                       | logging's captureWarnings method                                                                            |
| 29     | https://huggingface.co/docs/transformers/main/en/main_classes/optimizer_schedules | Pytorch                                                                           | PyTorch                                                                                                     |
| 30     | https://huggingface.co/docs/transformers/main/en/main_classes/trainer | TrainingArgument                                                                      | TrainingArguments                                                                                           |
| 31     | https://huggingface.co/docs/transformers/main/en/model_doc/swin2sr    | Swin2R improves                                                                       | Swin2SR improves                                                                                            |
| 32     | https://huggingface.co/docs/transformers/main/en/model_doc/phobert    | EART                                                                                  | BART                                                                                                        |
| 33     | https://huggingface.co/docs/transformers/main/en/installation         | TensorFLow                                                                            | TensorFlow                                                                                                  |

**_(4)Broken Link_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 34     | https://huggingface.co/docs/transformers/main/en/community            | https://github/m3hrdadfi/soxan/blob/main/notebooks/Emotion_recognition_in_Greek_speech_using_Wav2Vec2.ipynb | https://github.com/m3hrdadfi/soxan/blob/main/notebooks/Emotion_recognition_in_Greek_speech_using_Wav2Vec2.ipynb |

**_(5)Omissions in Text:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 35     | https://huggingface.co/docs/transformers/main/en/peft                 | or about the 🤗 PEFT library                                                          | or learn about the 🤗 PEFT library                                                                           |
| 36     | https://huggingface.co/docs/transformers/main/en/model_doc/clipseg    | zero- and one-shot image segmentation                                                 | zero-shot and one-shot image segmentation                                                                    |
| 37     | https://huggingface.co/docs/transformers/main/en/autoclass_tutorial   | feature extractor processes the audio signal the correct input format                 | feature extractor processes the audio signal into the correct input format                                   |
| 38     | https://huggingface.co/docs/transformers/main/en/pad_truncation       | The three arguments you need to are                                                   | The three arguments you need to know are                                                                     |
| 39     | https://huggingface.co/docs/transformers/main/en/tasks/video_classification | was obtained fine-tuning on a similar downstream                                  | was obtained by fine-tuning on a similar downstream                                                          |
| 40     | https://huggingface.co/docs/transformers/main/en/tasks_explained      | most models are a variant of an encoder, decoder, or encoder-decoder structure         | most models are a variant of an encoder, a decoder, or an encoder-decoder structure                          |
| 41     | https://huggingface.co/docs/transformers/main/en/main_classes/logging | This the default level                                                                | This is the default level                                                                                    |
| 42     | https://huggingface.co/docs/transformers/main/en/tasks/text-to-speech | example from the test dataset obtain a speaker                                        | example from the test dataset to obtain a speaker                                                            |

**_(6)Punctuations Based Typos:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 43     | https://huggingface.co/docs/transformers/main/en/model_doc/mixtral    | 45 billion parameters,,                                                               | 45 billion parameters,                                                                                      |
| 44     | https://huggingface.co/docs/transformers/main/en/model_doc/xlm-roberta| XLM-Ris                                                                               | XLM-R is                                                                                                     |
| 45     | https://huggingface.co/docs/transformers/main/en/accelerate           | Accelerate's [~accelerate.Accelerator.backward]method                                 | Accelerate's [~accelerate.Accelerator.backward] method                                                       |

**_(7)Minor Text Issues:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 46     | https://huggingface.co/docs/transformers/main/en/quantization/aqlm    | quantizes multiple weights together and take advantage                                | quantizes multiple weights together and takes advantage                                                     |
| 47     | https://huggingface.co/docs/transformers/main/en/main_classes/logging | respect the following heuristic                                                      | respect the following heuristics                                                                            |
| 48     | https://huggingface.co/docs/transformers/main/en/performance          | optimal setting                                                                      | optimal settings                                                                                            |
| 49     | https://huggingface.co/docs/transformers/main/en/sagemaker            | Table of Content                                                                     | Table of Contents                                                                                           |
| 50     | https://huggingface.co/docs/transformers/main/en/perf_train_gpu_one   | 200 millions parameters                                                              | 200 million parameters                                                                                      |
| 51     | https://huggingface.co/docs/transformers/main/en/model_sharing        | commit history as well as the difference                                             | commit history as well as the differences                                                                   |
| 52     | https://huggingface.co/docs/transformers/main/en/bertology            | while extract information                                                            | which extracts information                                                                                  |
| 53     | https://huggingface.co/docs/transformers/main/en/benchmarks           | Hereby                                                                               | Here                                                                                                        |
| 54     | https://huggingface.co/docs/transformers/main/en/quantization/bitsandbytes | save an addition 0.4 bits/parameter                                               | save an additional 0.4 bits/parameter                                                                        |
| 55     | https://huggingface.co/docs/transformers/main/en/trainer              | To pass extra arguments supports by GaLore                                           | To pass extra arguments supported by GaLore                                                                  |
| 56     | https://huggingface.co/docs/transformers/main/en/benchmarks           | so that benchmark results on their own are not very useful for the community         | as a consequence, benchmark results on their own are not very useful for the community                       |
| 57     | https://huggingface.co/docs/transformers/main/en/tasks/asr            | You can quickly load a evaluation method                                             | You can quickly load an evaluation method                                                                    |
| 58     | https://huggingface.co/docs/transformers/main/en/tasks/audio_classification | You can quickly load a evaluation method                                        | You can quickly load an evaluation method                                                                    |
| 59     | https://huggingface.co/docs/transformers/main/en/tasks/prompting      | Chain-of-though                                                                     | Chain-of-thought                                                                                             |
| 60     | https://huggingface.co/docs/transformers/main/en/model_doc/openai-gpt | will long range dependencies                                                        | with long range dependencies                                                                                 |
| 61     | https://huggingface.co/docs/transformers/main/en/debugging            | model inhibits                                                                      | model exhibits                                                                                               |
| 62     | https://huggingface.co/docs/transformers/main/en/model_doc/code_llama | cast the load the checkpoints                                                       | load the checkpoints                                                                                         |
| 63     | https://huggingface.co/docs/transformers/main/en/model_doc/mpt        | MPT models is a series                                                              | MPT models are a series                                                                                      |
| 64     | https://huggingface.co/docs/transformers/main/en/tasks/image_text_to_text | optimized to fit in smaller hardware                                             | optimized to fit on smaller hardware                                                                         |
| 65     | https://huggingface.co/docs/transformers/main/en/preprocessing        | Whether your data is text, images, or audio, they need to be converted               | Whether your data is text, images, or audio, it needs to be converted                                        |

**_(8)Formatting Issues:_**
| Sr No. | URL                                                                  | Typo                                                                                  | Correction                                                                                                  |
|--------|-----------------------------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
| 66     | https://huggingface.co/docs/transformers/main/en/perf_train_tpu_tf    | model.fit()                                                                           | `model.fit()`                                                                                               |
| 67     | https://huggingface.co/docs/transformers/main/en/perf_hardware        | openai-community/gpt2                                                                 | `openai-community/gpt2`                                                                                     |

I have created a PR regarding the same. If there are any changes to be made, please do let me know. Hope This helps 🤗.

### PR DESCRIPTION
**Pull request by nnilayy**
# What does this PR do?

Fixes #33349 
This PR fixes majority of the typos in the `docs` folder for `transformers[en]` documentation. 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

CC: @stevhliu 🤗

### CODE REVIEW

File: docs/source/en/main_classes/logging.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ transformers.logging.set_verbosity_info()
 ```
 
 You can also use the environment variable `TRANSFORMERS_VERBOSITY` to override the default verbosity. You can set it
-to one of the following: `debug`, `info`, `warning`, `error`, `critical`. For example:
+to one of the following: `debug`, `info`, `warning`, `error`, `critical`, `fatal`. For example:
```
Comment (LysandreJik) [inline docs/source/en/main_classes/logging.md:33]: I don't think we have `fatal`, where is this coming from?

File: docs/source/en/peft.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Now you can use [`~peft.PeftModel.set_adapter`] to set which adapter to use:
 ```py
 # use adapter_1
 model.set_adapter("adapter_1")
-output = model.generate(**inputs)
+output_disabled = model.generate(**inputs)
```
Comment (LysandreJik) [inline docs/source/en/peft.md:128]: Are you sure about this one?

**Review by LysandreJik — APPROVED** [review]
That's a lot of typo fixes, thanks!

File: docs/source/en/peft.md (hunk 2 above)
Comment (nnilayy) [reply docs/source/en/peft.md:128]: https://github.com/huggingface/transformers/blob/a70286f827ea46573e498fbc185a2498bc67d709/docs/source/en/peft.md?plain=1#L126-L134

Yup, in here `output_disabled[0]` has been referenced but never declared in the file, and looking at the code below using the pretty much the same code block for `output_enabled`, I assume its typo, where `output` should have been `output_disabled`.

File: docs/source/en/main_classes/logging.md (hunk 1 above)
Comment (nnilayy) [reply docs/source/en/main_classes/logging.md:33]: https://github.com/huggingface/transformers/blob/a70286f827ea46573e498fbc185a2498bc67d709/src/transformers/utils/logging.py#L168-L178

Currently using `transformers-4.44.2` in a Kaggle Notebook, I am able to generate the logs with `logger.fatal("FATAL")`.

image

Is there any plan to deprecate this functionality? If not, then I apologize for the confusion, I didnt see that `critical` and `fatal` have the same logging level and the fixed commit should have been:
`debug, info, warning, error, critical/fatal`

If `fatal` is going to be deprecated, i'll update the PR and fix it, do let me know if that's the case. Thanks for Reviewing 🤗

Comment (LysandreJik) [reply docs/source/en/main_classes/logging.md:33]: No worries, let's keep it like that in the meantime
